Add fan speed mode support to Synology DSM (#45991)
home-assistant/home-assistant.github.io: source/_integrations/synology_dsm.markdown

Shutdown the NAS.

> ## Select entities

Reviewer: According to the integration [template](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown), this section should be:
```suggestion
## Selects
```

### Fan speed mode

> If your NAS supports it, you can control the system fan speed mode using the **Fan speed mode** select entity. Available modes are **Quiet mode**, **Cool mode**, and **Full-speed mode**.

Reviewer: According to the integration [template](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown), this should be written as:
```suggestion
- **Fan speed mode**
  - **Description**: The mode of the system fan speed, if your NAS supports it.
  - **Options**: Quiet mode, Cool mode, Full-speed mode.
```
